File: SDKs/Aspose.Slides_Cloud_SDK_for_CSharp/src/Com/Aspose/Slides/Model/FontSet.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Slides.Model {
  public class FontSet {
    public string ComplexScript { get; set; }

    public string EastAsian { get; set; }

    public string Latin { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class FontSet {\n");
      sb.Append("  ComplexScript: ").Append(ComplexScript).Append("\n");
      sb.Append("  EastAsian: ").Append(EastAsian).Append("\n");
      sb.Append("  Latin: ").Append(Latin).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Slides-Cloud-SDK-for-Java/src/main/java/com/aspose/slides/model/FontSet.java
package com.aspose.slides.model;

public class FontSet {
  private String ComplexScript = null;
  private String EastAsian = null;
  private String Latin = null;
  /**
	 * getComplexScript
	 * Gets String
	 * @return ComplexScript
	 */
  public String getComplexScript() {
    return ComplexScript;
  }

	/**
	 * setComplexScript
	 * Sets String
	 * @param ComplexScript String
	 */
  public void setComplexScript(String ComplexScript) {
    this.ComplexScript = ComplexScript;
  }

  /**
	 * getEastAsian
	 * Gets String
	 * @return EastAsian
	 */
  public String getEastAsian() {
    return EastAsian;
  }

	/**
	 * setEastAsian
	 * Sets String
	 * @param EastAsian String
	 */
  public void setEastAsian(String EastAsian) {
    this.EastAsian = EastAsian;
  }

  /**
	 * getLatin
	 * Gets String
	 * @return Latin
	 */
  public String getLatin() {
    return Latin;
  }

	/**
	 * setLatin
	 * Sets String
	 * @param Latin String
	 */
  public void setLatin(String Latin) {
    this.Latin = Latin;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class FontSet {\n");
    sb.append("  ComplexScript: ").append(ComplexScript).append("\n");
    sb.append("  EastAsian: ").append(EastAsian).append("\n");
    sb.append("  Latin: ").append(Latin).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

